cmd/kyma/alpha/deploy/opts.go

```diff
@@ -29,6 +35,51 @@ func NewOptions(o *cli.Options) *Options {
 	return &Options{Options: o}
 }
 
+func (o *Options) ResolveLocalWorkspacePath() (string, error) {
+
+	if o.WorkspacePath == "" {
+		o.WorkspacePath = defaultWorkspacePath
+	}
+	//resolve local Kyma source directory only if user has not defined a custom workspace directory
+	if o.Source == VersionLocal && o.WorkspacePath == defaultWorkspacePath {
+		//use Kyma sources stored in GOPATH (if they exist)
+		goPath := os.Getenv("GOPATH")
+		if goPath != "" {
```

Comment:
Maybe also deal with the situation when GOPATH == "". Maybe by returning an error, since this should block the installation at this point, since it will fail later anyway

Reply:
If goPath is not set then we return default workspace and try to use that as a workspace. This was the behaviour before but I am not sure what would be the nice UX because GOPATH in general is needed only for development purpose

Reply:
As discussed in meeting, we will stick to this behaviour for now